ads/google/a4a/google-data-reporter.js

```diff
@@ -103,7 +103,7 @@ function isInReportableBranch(ampElement, namespace) {
  * @param {string} namespace
  * @param {number|string} slotId A unique numeric identifier in the page for
  *    the given element's slot.
- * @return {!GoogleAdLifecycleReporter|!BaseLifecycleReporter}
+ * @return {!./performance.BaseLifecycleReporter}
  */
 export function getLifecycleReporter(ampElement, namespace, slotId) {
```

Comment:
Actually, do we still need this? Can we merge `getLifecycleReporter` and `googleLifecycleReporterFactory` to one function? (we don't have to do it now though)